spec/features/create_new_work_spec.rb

```diff
@@ -117,14 +117,25 @@
           fill_in 'Abstract', with: 'User provided abstract'
           check 'Oral history'
 
+          find('label.switch').click # Use auto-generated citation
```

Comment:
This test, which is supposed to test the feature "a user can create a work" is really stretching to test this validation.  I think this test is getting too complicated and I urge you to instead create a lighter weight request spec that tests just this single validation.  Then if it breaks, it's much easier to spot which part broke.

Reply:
Ok, reverted change to this feature spec and move the validation test to a new request spec